common/scala/src/main/scala/org/apache/openwhisk/core/database/ActivationFileStorage.scala

```diff
@@ -92,8 +93,11 @@ class ActivationFileStorage(logFilePrefix: String,
 
   private def transcribeActivation(activation: WhiskActivation, additionalFields: Map[String, JsValue]) = {
     val transactionType = Map("type" -> "activation_record".toJson)
-    val message = Map(
-      "message" -> s"Activation record '${activation.activationId}' for entity '${activation.name}'".toJson)
+    val message = if (writeResultToFile) {
```

Comment:
you can actually move the `if` inside the map but that's not necessary just fyi.